ci-operator/jobs/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master-presubmits.yaml

```diff
@@ -13,6 +13,7 @@ presubmits:
       ci-operator.openshift.io/prowgen-controlled: "true"
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-windows-machine-config-operator-master-aws-e2e-operator
+    optional: true
```

Comment:
`optional` implies that it will still run. There is another option to make it manual i.e. only run the test if someone issues `/test azure-e2e-operator`. Please use that option until we get Azure working.